### PR TITLE
feat: add confidential computing example for intel arch

### PR DESCRIPTION
**Pull request by arthurlapertosa**
add confidential computing example for intel arch

### CODE REVIEW

**Comment by arthurlapertosa** [comment]
@apeabody Can you run the build, please?

File: test/fixtures/confidential_intel_compute_instance/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  value       = module.confidential_computing_intel.self_link
+}
+
+output "name" {
+  description = "Name of the instance templates."
+  value       = module.confidential_computing_intel.name
+}
+
+output "instance_self_link" {
+  description = "Self-link for compute instance"
```
Comment (erlanderlo) [inline test/fixtures/confidential_intel_compute_instance/outputs.tf:28]: nit - missing a period at the end to match the rest of the descriptions

**Review by erlanderlo — APPROVED** [review]
lgtm; thanks for getting this example up to showcase our new integration

**Comment by arthurlapertosa** [comment]
@apeabody could you run the build again? Also, could you please review it whenever possible?

**Review by apeabody — APPROVED** [review]
Thanks for the contribution @arthurlapertosa!

Reviewed by @erlanderlo

**Pull request merged**